src/php/php_server.rs

```diff
@@ -32,6 +34,14 @@ impl PhpServer {
 pub(crate) fn start() -> PhpServer {
     let php_bin = binaries::get_project_version();
 
+    let phpbin_path = PathBuf::from(php_bin.as_str());
+
+    if !phpbin_path.is_executable() {
+        error!("PHP Binary not found or not executable: {}", php_bin.as_str());
+        info!("You can execute \"rymfony php:list --refresh\" to update binaries paths cache");
```

Comment:
```suggestion
        error!("You can execute \"rymfony php:list --refresh\" to update binaries paths cache.");
```

For now, let's keep error messages together. An informational message after an error one doesn't really make sense.

In the future, we'll have to find a proper way to throw errors and have a look similar to Symfony CLI or Symfony Console.